Showcase-iOS/Dashboard/Cell/ShowcaseAppCollectionViewCell.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ShowcaseAppCollectionViewCell.swift
+//  Showcase-iOS
+//
+//  Created by Edward Mtshweni on 2018/05/14.
+//  Copyright © 2018 DVT. All rights reserved.
+//
+
+import UIKit
+import Kingfisher
+
+class ShowcaseAppCollectionViewCell: UICollectionViewCell {
+    @IBOutlet private weak var imageView: UIImageView!
+    @IBOutlet private weak var shortDescriptionLabel: UILabel!
+    @IBOutlet private weak var clientLabel: UILabel!
+    
+    var firebaseStorage: FIRStoring?
+    
+    func populateCell(with showcaseViewModel: ShowcaseAppViewModel) {
+        self.shortDescriptionLabel.text = showcaseViewModel.shortDescription
+        self.clientLabel.text = showcaseViewModel.client
+        self.populateImageView(with: showcaseViewModel.iconUrl)
+    }
+    
+    func populateImageView(with iconUrl: String?) {
+        guard let imageUrl = iconUrl else {
+            return
+        }
+        guard let firStorage = self.firebaseStorage else  {
+            return
+        }
+        let imageFetcher = ImageFetcher(from: firStorage)
+        imageFetcher.fetchImage(imageUrl) { (url, error) in
```

Comment:
This logic needs to be in the viewModel,This will need to be refactored in the next pull request. Ill come and sit with you. ill merge this for now. but making a task to refactor because this logic shouldnt be here